servers/audio/audio_stream.cpp

```diff
@@ -497,6 +497,14 @@ float AudioStreamRandomizer::get_random_volume_offset_db() const {
 	return random_volume_offset_db;
 }
 
+float AudioStreamRandomizer::get_volume_variation_range() const {
```

Comment:
Nit: since this doesn't return the range, but rather the calculated random value, could we name it differently?